Added TCP proto support to the net mode tester

gygachad opened the pull request.

ifratric commented:
Hi, this is a fairly large patch, so could you explain what issue(s) it resolves? My understanding was that TCP was supported already (but perhaps it wasn't working correctly for some reason?)

gygachad commented:
Hi! Not exactly. [There](https://github.com/googleprojectzero/winafl/blob/31c0b17aed592f429c2ded00fbe315dcee8bc52c/test_netmode.cpp#L41) was no support for TCP, so I decided to add it. I am trying to start fuzz Windows services now with WinAFL. First of all I added TCP proto to custom_net_fuzzer. Then I added server part for testing. Also I added argument with a port number.
